test/unit/range/decorator/gap_decorator_test.cpp

```diff
@@ -21,12 +21,13 @@
 #include "../iterator_test_template.hpp"
 #include "../../alignment/aligned_sequence_test_template.hpp"
 
-using decorator_t = gap_decorator<std::vector<dna4> const &>;
+using decorator_t = seqan3::gap_decorator<std::vector<seqan3::dna4> const &>;
 
-std::vector<dna4> const dummy_obj{}; // dummy lvalue for type declaration of views
-using decorator_t2 = gap_decorator<
+std::vector<seqan3::dna4> const dummy_obj{}; // dummy lvalue for type declaration of views
+using decorator_t2 = seqan3::gap_decorator<
                          decltype(std::ranges::subrange<decltype(dummy_obj.begin()),
-                                                 decltype(dummy_obj.begin())>{dummy_obj.begin(), dummy_obj.end()})>;
+                                                        decltype(dummy_obj.begin())>{dummy_obj.begin(),
+                                                                                     dummy_obj.end()})>;
```

Comment:
Here it seems beneficial to define a type
```cpp
using iter_t = decltype(dummy_obj.begin());
```
before to shorten this. However, this is just an aesthetical suggestion. 💅